app/scripts/components/common/blocks/scrollytelling/index.tsx

```diff
@@ -430,58 +415,50 @@ function Scrollytelling(props) {
           </CSSTransition>
         </SwitchTransition>
 
-        <Styles>
-          <Basemap />
+        <Map
```

Comment:
Oh I see that only the components are replaced, and work around how the data is shaped through a utilify function(`reconcileVizDataset`). Now that I am deep in rewriting `s-explore`, I can see how this approach can help break down the task better.  On the other hand, I think we need to deprecate  `useAsyncLayers` sooner since it unnecessarily diverges how we reconcile stac data with our layer data. 

How do you think about deprecating `useAsyncLayers` ?

Reply:
Thanks for pointing that! I'm definitely in favor of deprecating unnecessary diverging code, but just to clarify: `useAsyncLayers` accepts an array of `datasetId` & `layerId` (taken from the `Chapter's` from the MDX, validates it and then calls `useLayersInit` with the layer ids. Do you maybe meant that we could deprecate `useLayersInit` instead by switching to `useReconcileWithStacMetadata` (which I think is definitely something we could & should do)?

Reply:
Yes! I meant deprecating all the old methods including `useLayersInit` , and replacing it with `useReconcileWithStsacMetadata`!